temporalio/bridge/src/worker.rs

```diff
@@ -52,6 +51,39 @@ pub struct WorkerConfig {
     nondeterminism_as_workflow_fail_for_types: HashSet<String>,
 }
 
+#[derive(FromPyObject)]
+pub struct TunerHolder {
+    workflow_slot_supplier: SlotSupplier,
+    activity_slot_supplier: SlotSupplier,
+    local_activity_slot_supplier: SlotSupplier,
+}
+
+#[derive(FromPyObject)]
+pub enum SlotSupplier {
+    FixedSize(FixedSizeSlotSupplier),
+    ResourceBased(ResourceBasedSlotSupplier),
+}
+
+#[derive(FromPyObject)]
+pub struct FixedSizeSlotSupplier {
+    num_slots: usize,
+}
+
+#[derive(FromPyObject)]
+pub struct ResourceBasedSlotSupplier {
+    minimum_slots: usize,
+    maximum_slots: usize,
+    // Need pyo3 0.21+ for this to be std Duration
+    ramp_throttle_ms: u64,
```

Comment:
```suggestion
    ramp_throttle_millis: u64,
```

We do this in plenty of other places, it's fine for the bridge (we usually use `_millis` suffix)

Reply:
Sure, it'd be nicer if it just worked though